.github/CONTRIBUTING.md

```diff
@@ -0,0 +1,125 @@
+
+If you made complex changes, it is helpful to also include comments next to your code, in order to document the changes for reviewers and other contributors.
+
+We are using pre-commit hooks, to stick to a nice structure and formatting. See [pre-commit](https://pre-commit.com/) and [git hooks](https://git-scm.com/book/en/v2/Customizing-Git-Git-Hooks).
+
+Once your changes are ready, you can commit and push them to github.
+1. To commit all modified files into the local copy of your repo, do `git commit -am 'A commit message'`.
+2. To push the changes up to your forked repo on GitHub, do a `git push`.
+
+### Pull Request
+
+Once you’re ready or need feedback on your code, open a Pull Request, also known as a PR, on the github project page.
+- Don't forget to [link PR to issue](https://github.com/Open-EO/openeo-processes-dask/issues) if you are solving one.
+- Once you submit your PR, an openeo-processes-dask team member will review your changes. We may ask questions or request additional information.
+- We may ask for changes to be made before a PR can be merged.
+- As you update your PR and apply changes, mark each conversation as [resolved](https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/commenting-on-a-pull-request#resolving-conversations).
+- If you run into any merge issues, checkout this [git tutorial](https://github.com/skills/resolve-merge-conflicts) to help you resolve merge conflicts and other issues.
+
+You can directly comment in the PR if you want the reviewer to pay particular attention to something - If your request is not ready to be merged, just say so in your pull request message and create a “Draft PR". This way, you can get preliminary code review and others can see what is currently being worked on.
+
+## New releases
+
+Once the reviewer(s) approve the PR and there are no more changes requested, the PR will be merged into the main branch. A new release will then be created based on the main branch.
+
+There will be a new release at least every two weeks. (In case, there were no new changes at all, the release might be skipped.)
+
+If important changes are added - such as bug fixes and additional processes - new releases might be made in between.
+
+Small changes - such as new comments, updated documentation - will be included in the bi-weekly releases.
+
+## Adding new processes
+
+If you only want to update implementation details in a process, the specification should remain as it is and you do not need to update the submodule.
+
+If you want to add a new process or update the parameters of the process, you will also need to interact with the submodule.
+
+The specifications come from a fork of the official openeo-processes: https://github.com/eodcgmbh/openeo-processes
+
+To add a new process:
+- add the specification to https://github.com/eodcgmbh/openeo-processes
```

Comment:
Maybe specify: create a fork, add your process to the root folder, moving it from the missing-processes folder or creating a new process definition.